react-native-in-app-payments-quickstart/android/build.gradle

```diff
@@ -2,18 +2,20 @@
 
 buildscript {
     ext {
-        buildToolsVersion = "30.0.3"
+        buildToolsVersion = "33.0.2"
         minSdkVersion = 24
-        compileSdkVersion = 32
-        targetSdkVersion = 32
+        compileSdkVersion = 33
+        targetSdkVersion = 33
         ndkVersion = "20.1.5948944"
-        sqipVersion="1.5.6"
+        sqipVersion="1.6.1"
         kotlin_version = '1.7.10'
     }
     repositories {
         google()
-        jcenter()
         mavenCentral()
+         maven {
+            url "https://maven.google.com"
+        }
```

Comment:
This is not needed if you have `google()`, as far as I know

Reply:
Done

Reply:
Done? The `maven { url "https://maven.google.com/" }` is still here, I don't understand..

Reply:
Removed !